plugin/public/css/block.css

```diff
@@ -1,20 +1,22 @@
-.folder {
-  background-color: rgb(197, 218, 224);
-  font-weight: 500;
+.sgdd-block-settings-folder, .sgdd-block-settings-folder-up {
+	color: var(--wp-admin-theme-color);
```

Comment:
Is this variable provided by WordPress? I couldn't find any documentation for it...

If so, would it be available on older WordPress versions?

Reply:
This CSS variable is connected with WordPress Admin Color Scheme and it was included to core with WordPress 3.8.

Reply:
Oh, but it just occured to me that there is no preprocessing done on the CSS so this would break on IE :/ COuld we please use something with better compatibility?

Reply:
I can add fallback color but IE is not fully supported by WordPress (when you enter to WordPress admin panel it advice you change browser to more secure). Apart this, when I try to edit page with this plugin in IE (I was surprised that I still have this browser installed) I see message that "this webpage doesn't support block "Files and Docs from Google Drive" - so anyway it is not working on admin side in IE.

Reply:
Well, then let's not bother if it doesn't work anyway :(